components/dockmanager/overview.md

```diff
@@ -0,0 +1,256 @@
+                        <strong>Recent Activity</strong>
+                    </HeaderTemplate>
+                    <Content>
+                        <p>User A updated Project X</p>
+                        <p>User B commented on Task 3</p>
+                        <p>New file uploaded for Task 2</p>
+                    </Content>
+                </DockManagerContentPane>
+
+            </Panes>
+        </DockManagerSplitPane>
+
+        <DockManagerSplitPane>
+            <Panes>
+                <DockManagerTabGroupPane>
+                    <Panes>
+
+                        <DockManagerContentPane>
+                            <HeaderTemplate>
+                                <strong>Project Overview</strong>
+                            </HeaderTemplate>
+                            <Content>
+                                <p>Summary of ongoing projects, completion rate, and deadlines.</p>
+                            </Content>
+                        </DockManagerContentPane>
+
+                        <DockManagerContentPane>
+                            <HeaderTemplate>
+                                <strong>Team Performance</strong>
+                            </HeaderTemplate>
+                            <Content>
+                                <p>Performance metrics of different teams, including task completion and efficiency.</p>
+                            </Content>
+                        </DockManagerContentPane>
+
+                    </Panes>
+                </DockManagerTabGroupPane>
+
+                <DockManagerContentPane>
+                    <HeaderTemplate>
+                        <strong>Notifications</strong>
+                    </HeaderTemplate>
+                    <Content>
+                        <p>New messages, mentions, and important updates.</p>
+                    </Content>
+                </DockManagerContentPane>
+
+            </Panes>
+        </DockManagerSplitPane>
+    </DockManagerPanes>
+
+    <DockManagerFloatingPanes>
+        <DockManagerSplitPane>
+            <Panes>
+
+                <DockManagerContentPane>
+                    <HeaderTemplate>
+                        <strong>Team Chat</strong>
+                    </HeaderTemplate>
+                    <Content>
+                        <p>Live conversation with team members.</p>
+                        <TelerikTextBox @bind-Value="@ChatMessage"></TelerikTextBox>
+                    </Content>
+                </DockManagerContentPane>
+
+            </Panes>
+        </DockManagerSplitPane>
+    </DockManagerFloatingPanes>
+</TelerikDockManager>
+
+@code {
+    private string ChatMessage { get; set; }
+}
+````
+
+## State
+
+The [Dock Manager allows getting and setting its state](slug://dockmanager-state). The DockManager state contains information about the panes hierarchy, a collection of all the floating panes, and all the DockManager configurations, such as its orientation.
+
+## Dock Types
```

Comment:
"Dock Types" implies that "dock" is a noun, while it's a verb. Consider "Docking Types", similar to the heading in the linked article.